examples/grpc/src/main/java/example/armeria/grpc/Main.java

```diff
@@ -69,7 +69,8 @@ static void configureServices(ServerBuilder sb) {
                                                    "BlockingHello", exampleRequest)
                                   .exclude(DocServiceFilter.ofServiceName(
                                           ServerReflectionGrpc.SERVICE_NAME))
-                                  .build());
+                                  .build())
+          .verboseResponses(false);
```

Comment:
Perhaps we can remove this to keep our examples simple enough?

Reply:
Oh, maybe I set it before because `HelloServiceTest` failed in the build. 🤔 

https://github.com/line/armeria/pull/4171/commits/84c8b9569d1ff6d97b0e16d1384de6aefa986e02

Reply:
Oops, wrong link above: https://github.com/line/armeria/pull/4171/commits/806e47a540b15a8ce22274476ae510ac5866f8c0 (`HelloServiceTest` failed here.)
Let me take a look again 🤔